### PR TITLE
feat(font-size): add font size with line-height and letter-spacing

### LINKED ISSUE #145: [Font Size] Update & New
**Problem**
Need a medium size font between the existing `display-xl`(64px) and `display-lg`(32px) font size.

**Solution**
Change the exisiting `display-xl` -> `display-2xl `and Add new size(48px) font to` display-xl`.

**Figma / Jira Link**
https://www.figma.com/file/DVwz3WQLLIxWXj4d4W8MQI/Foundation?node-id=17863%3A182708&t=D23wupnkfCCsR66g-1

**Additional Description**
Font weights are same with `display-xl`(regular, medium, bold)

### PR DESCRIPTION
**Pull request by song2kim**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [X] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [X] Updated Storybook documents
- [ ] Tested with console(if usages are changed) 

### Description
* Update tailwindcss package version from 1.2.0 to 1.9.6
* Add semantic fontSize with default line-height and letter-spacing

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -183,7 +183,7 @@
     "style-loader": "^1.0.0",
     "stylelint": "^13.2.1",
     "stylelint-config-standard": "^20.0.0",
-    "tailwindcss": "^1.2.0",
+    "tailwindcss": "^1.9.0",
```
Comment (minseolee) [inline package.json]: Version of tailwindcss in console is, `"^1.9.6"`.
How about aligning it ?!

**Review by minseolee — APPROVED** [review]
LGTM!
Left a tinytiny comment

File: package.json (hunk 1 above)
Comment (song2kim) [reply package.json]: I think it your suggestion is good!!! I'll do

File: src/foundation/typography/PTypography.stories.mdx [hunk 2]
```diff
@@ -26,19 +25,55 @@ export const Template = (args, { argTypes }) => ({
...
+            <td>500</td>
+            <td class="font-medium">ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz12345678910</td>
+        </tr>
+        <tr>
+            <td>font-normal</td>
+            <td>400</td>
+            <td class="font-normal">ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz12345678910</td>
+        </tr>
+    </tbody>
+</table>
```
Comment (WANZARGEN) [inline src/foundation/typography/PTypography.stories.mdx:76]: praise: Nice documentation!!!

**Review by WANZARGEN — APPROVED** [review]
LGTM! What a beautiful PR.... 😍

**Pull request merged**